src/index.js

```diff
@@ -121,9 +109,12 @@ const sendBookingData = (inputBookingData) => {
       console.log(error.number)
       console.log(error.message)
     })
+
+  updateBookingData()
```

Comment:
I don't think you need this function or to call it here. The function is just repeating getBookingsData, which you are also calling in createCurrentDataSet(). This might be why you aren't getting the updated data all of the time.